wiki/BanchoBot/pt-br.md

```diff
@@ -1,40 +1,160 @@
+*Para outros usos, veja [Bancho (desambiguação)](/wiki/Disambiguation/Bancho).*
 
-O BanchoBot é um bot (um conjunto de comandos com respostas automatizadas) programado para ajudar as pessoas na conversa exibindo certas informações e anunciando mensagens relacionadas ao jogo. O BanchoBot é seu host do servidor do [osu!chat](/wiki/Community/Internet_Relay_Chat) e tem um perfil chamado *[BanchoBot](https://osu.ppy.sh/users/3)*.
+![Perfil de usuário do BanchoBot](img/BanchoBot.jpg "BanchoBot's user card")
 
-## Utilizando o BanchoBot
+**BanchoBot** (às vezes chamado de *Bancho*) é um chat bot online feito para o osu! que ajuda os jogadores pelo chat dentro do jogo, anunciando mensagens relacionadas a ele (exemplo: vezes totais jogadas, tentativas, etc..), e respondendo a certos comandos . Ele foi programado pelo [Echo](https://osu.ppy.sh/users/431) e é o fornecedor do [Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat).
 
-Para abrir o BanchoBot, digite `!help` (ou qualquer um dos comandos abaixo) no seu console de conversa (qualquer aba serve).
-Assim que fizer isso, a aba do BanchoBot será aberta.
-
-Fazer isso irá mostrar uma lista de comandos (os da tabela abaixo).
+BanchoBot também tem seu próprio [perfil osu!](https://osu.ppy.sh/users/3) e [conta no Twitter](https://twitter.com/banchoboat).
 
 ## Comandos
 
-Você pode usar `!<comando>` ou `/bb <comando>` em qualquer aba de conversa.
+*Para uma lista de comandos em jogo, veja: [Consolde do Chat](/wiki/Client/Interface/Chat_console#commands-list)*
+
+O BanchoBot pode responder jogadores através de comandos específicos digitados no chat. Todos os comandos do BanchoBot começam com um ponto de exclamação (`!`) seguidos pelo nome do comando dependente. (com espaço entre eles). Esses comandos podem ser usados em canais de chat através de mensagens privadas com o BanchoBot.
+
+Se  um  usuário normal enviar um comando no chat público, outros usuários não irão ver, e a resposta será mostrada em uma mensagem privada com o BanchoBot. Usuários também podem usar o comando `/bb` no cliente dentro do jogo para automaticamente abrir uma nova aba com o BanchoBot e enviar um comando imediatamente.
+
+A lista de todos os comandos do BanchoBot podem ser encontrdas abaixo:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` mostra uma lista de todos os comandos disponíveis dos BanchoBot. Um exemplo de quando esse comando é usado aparece abaixo:
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <usário>
+13:00 BanchoBot: STATS <usuário>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <motivo> - chame um administrador
+13:00 BanchoBot: REQUEST [lista] - aparece uma solicitação aleatória recente de mod
+13:00 BanchoBot: ROLL <número> - rola um dado e obtenha um resultado aleatório de 1 para o número (padrão 100)
```

Comment:
don't translate this, as the comment below says. this shows what the `!help` command displays in the chat